hopsworks-api/src/main/java/io/hops/hopsworks/api/admin/SystemAdminService.java

```diff
@@ -68,8 +77,8 @@
    * @return
    * @throws AppException
    */
-  @POST
-  @Path("/changeMasterEncryptionPass")
+  @PUT
+  @Path("/encryption_pass/change")
```

Comment:
@kouzant @augustbonds the "change" is implicit by it being a PUT operation. We can remove it I think

Reply:
Yep, should be "PUT /encryptionPass" with response NO_CONTENT as you have specified.